<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::PermissionBarWidget</class>
 <widget class="QWidget" name="Otter::PermissionBarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>46</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="permissionLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="iconLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>32</horstretch>
       <verstretch>32</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="messageLabel"/>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QComboBox" name="permissionComboBox">
     <item>
      <property name="text">
       <string>Allow this time</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Always allow</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Always deny</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
